pulsar/consumer_partition.go

```diff
@@ -738,7 +738,7 @@ func createEncryptionContext(msgMeta *pb.MessageMetadata) *EncryptionContext {
 
 func (pc *partitionConsumer) ConnectionClosed() {
 	// Trigger reconnection in the consumer goroutine
-	pc.log.Debug("connection closed and send to connectClosedCh")
+	pc.log.Warnf("connection closed and send to connectClosedCh")
```

Comment:
It doesn't look like a WARN behavior here, INFO might be a reasonable log level

Reply:
Resolved.